### PR TITLE
2016 regulations-Korean translation

### PR DESCRIPTION
**Pull request by rnjs37gus**
2016 version of Korean WCA regulation

### CODE REVIEW

File: korean/wca-regulations.md [hunk 1]
```diff
@@ -100,30 +105,41 @@ WCA 규정과 지침의 변천은 [GitHub](https://github.com/cubing/wca-documen
...
+    - 3d1) 의학적으로 입증된 시각장애가 있는 선수들에겐 다음 예외규정이 적용된다.
+        - 3d1a) 맹인 참가자들은 각 면에 다른 재질이 사용된 큐브를 사용할 수 있다. 각 면은 스크램블과 심판을 위해 구분되는 색이 있어야 한다.
+        - 3d1b) 꼭 필요한 수의 색을 구별하지 못하는 색맹 참가자들은 [규정 2s항](regulations:regulation:2s)에 따라 명백히 인정받았다면 문양을 사용할 수 있다. 문양은 스티커나 그림으로 만들 수 있다.
+    - 3d2) 퍼즐의 색은 한 면에 한 색이어야 하며, 각 면의 색은 일정하여야 한다. 또한 각각의 색은 다른 색들과 확연히 구분되어야 한다.
+    - 3d3) 퍼즐의 색칠된 부분이 스티커, 타일인 경우 두께가 절대로 1.5mm를 넘을 수 없다. 예외: 메가밍크스 조각은 WCA 파견위원의 결정 하에 1.5mm보다 약간 두꺼울 수 있
+- 3h) 퍼즐의 기본적인 기능을 향상시키는 어떠한 개조도 허용되지 않는다. 개조된 퍼즐은 개조가 개조되지 않은 퍼즐에 비해 선수에게 어떠한 추가적 정보를(예: 보이지 않는 조각의 판단) 주지 않는 한도 내에서만 인정된다.
     - 3h1) "필로우“ 버전의 큐브는 인정되지 않는다. 예외: 필로우 7x7x7 큐브는 허용된다.
-    - 3h2) "자체 색“ 큐브, 그리고 큐브의 내부에서 면의 색이 보이는 큐브들은 인정되지 않는다.
+    - 3h2) 큐브의 내부에서 면의 색이 보이는 큐브들(예: "자체 색" 큐브)는 허용된다. 이것은 다음 퍼즐들을 포함하지 않는다.
+        -3h2a) 투명한 조각이 있는 퍼즐. 예외: 오버레이 스티커([규정 3l항](regulations:regulation:3l) 참조)
```
Comment (viroulep) [inline korean/wca-regulations.md]: A space is missing here, it should be "- 3h2a".

File: korean/wca-regulations.md [hunk 2]
```diff
@@ -133,10 +149,19 @@ WCA 규정과 지침의 변천은 [GitHub](https://github.com/cubing/wca-documen
 - 4b) 퍼즐은 무작위로 컴퓨터로 생성된 스크램블 공식으로 스크램블 되어야 한다.
     - 4b1) 생성된 스크램블 공식들은 대회 전 미리 공개되어서는 안 되며, 어떠한 방법으로든 WCA 파견위원으로부터 골라져서는 안 된다.
     - 4b2) 스크램블 공식들은 해당하는 라운드가 끝나기 전까진 WCA 파견위원과 스크램블러들에게만 공개되어야만 한다. 예외: 최소회전종목에서는, 선수들이 라운드 도중 스크램블 공식을 받는다.([제 E 조](regulations:article:E) 참조)
+    - 4b3) 스크램블 프로그램: 스크램블 공식은 맞추는 데 2회전 이상 필요한 무작위 상태(각 상태는 같은 확률을 가짐)를 만들어야 한다. 아래의 추가/예외사항이 적용된다:
+        - 4b3a) 눈가리기 종목에서 스크램블 공식은 퍼즐 조각의 방향을 무작위(각 방향이 같은 확률을 가짐)로 만들어야 한다.
+        - 4b3b) 2x2x2 큐브: (무작위)상태는 맞추는데 적어도 4회전 이상이 필요하다.
+        - 4b3c) 스큐브: (무작위)상태는 맞추는데 적어도 7회전 이상이 필요하다.
+        - 4b3d) 스퀘어-1: (무작위)상태는 맞추는데 적어도 11회전 이상이 필요하다.
+        - 4b3e) 5x5x5, 6x6x6, 7x7x7 및 메가밍크스: 충분히 많은 무작위 회전(무작위 상태 대신), 맞추는데 적어도 2회전 이상 필요하다.
+    -4b4) 스크램블 공식은 최대 2시간까지만 작용되어야 한다. 이 시간은 스크램블 공식이 처음 적용되었을 때 시작된다.
```
Comment (viroulep) [inline korean/wca-regulations.md]: Same here it should be "- 4b4".

File: korean/wca-regulations.md [hunk 3]
```diff
@@ -332,64 +356,70 @@ WCA 규정과 지침의 변천은 [GitHub](https://github.com/cubing/wca-documen
...
+    - A1c) 어떤 종목에 참가하는 선수는 그 종목의 필요조건 (예: 그 퍼즐을 맞출 줄 아는 것)을 만족해야만 한다. 선수는 DNF나 부진한 기록을 노리고 참가해선 안된다. 어길 시 벌칙: WCA 파견위원의 결정에 따라 시도의 실격(DNF) 또는 종목의 실격([규정 2j항](regulations:regulation:2j) 참조)
 - A2) 스크램블:
     - A2a) 라운드에 불렸을 때, 선수는 퍼즐을 맞추어진 상태로 스크램블러에게 넘긴 뒤, 기록 측정을 위해 불려 질 때까지 정해진 구역에서 기다려야 한다.
     - A2b) 스크램블러는 [제 4 조](regulations:article:4) 규정에 따라 퍼즐을 섞는다.
+        - A2b1) 스퀘어-1 종목에서 운영진은 시도의 시작 전의 의도치 않은 이동을 방지하기 위해 큐브에 앏은 물체를 사용할 수 있다. 물체를 사용하는 경우, 라운드 시작 전에 이를 공지해야 한다.
     - A2c) 스크램블러가 스크램블을 시작하면 선수는 미리 보기 시간 전까지 퍼즐을 보아서는 안 된다.
         - A2c1) 스크램블러는 섞인 퍼즐 위에 덮개를 씌워 어떤 선수나 관중도 큐브의 어떤 부분이라도 볼 수 없게 한다. 덮개는 시도의 시작까지 덮여있어야 한다.
     - A2d) 스크램블러에게 큐브를 건네 받을 때, 심판은 퍼즐이 충분히 섞였는지를 간단하게 점검한다. 의심스러운 경우에 심판은 스크램블러에게 더 자세한 확인을 위해 연락한다.
-    - A2e) 심판은 퍼즐을 임의의 방향으로 스택맷 위에 놓고, 확실히 덮개로 덮였는지 확인한다. 선수는 특정한 방향을 요구할 수 없다.
+        -A2d1) 운영진은 스크램블러에게 정확한 스크램블 공식을 적용했는지 확인하기 위해 추가 절차를 요구할 수 있다. (예: 정확한 번호의 스크램블을 그림과 일치하게 적용했음을 확인하는 서명을 기록지에 하는 것, 혹은 '스크램블 검사자'를 두는 것)
```
Comment (viroulep) [inline korean/wca-regulations.md]: Same here, it should be "- A2d1".

**Comment by viroulep** [comment]
Thanks!
Beside the few syntax typos that I commented on, all looks good!

**Comment by rnjs37gus** [comment]
Thank you so much!

**Comment by viroulep** [comment]
Thanks, although I'm sorry I forgot to check everything :s
Translations for 4b3f and F3a are missing, could you please add them? :)

**Comment by rnjs37gus** [comment]
I should have checked my translations. I didn't know I missed those lines. Thank you!

**Comment by viroulep** [comment]
No problem!

Thanks, I'm going to merge both.

**Pull request merged**